### ISSUE TITLE
[Suggestion] Remove brackets around verse quotations

### ISSUE THREAD

**Issue by matthieugrieger** [Team-VerseBot/versebot#5]
Hey everyone,

I think it might be a good idea to remove the requirement for verse quotations to be surrounded by brackets ([]). This is a remnant from the time when VerseBot scanned every comment on reddit, and did not check username mentions. Now that the username mention system exists I don't think it would hurt anything to remove the bracket requirement.

What do you guys think?

**Comment by adamgrieger** [comment]
I think that removing it could work. Does VerseBot currently use RegEx to grab the quotations between the brackets? That's the only thought I have. If it is way easier to use RegEx, than the brackets would not really be a problem.

**Comment by matthieugrieger** [comment]
Yeah, it is part of the regex here: https://github.com/Team-VerseBot/versebot/blob/master/versebot/regex.py#L14-L15

If you just remove the beginning and end bits that are in parentheses I think that is the only change that needs to be made.

**Comment by adamgrieger** [comment]
Oh, that's not bad at all. I could look into that.
